apps/advisor/pkg/app/checkscheduler/checkscheduler.go

```diff
@@ -113,7 +116,6 @@ func (r *Runner) Run(ctx context.Context) error {
 			}
 			ticker.Reset(nextSendInterval)
 		case <-ctx.Done():
-			r.markUnprocessedChecksAsErrored(ctx, logger)
```

Comment:
This was not working, since the `ctx` is typically canceled when shuting down, it was not doing the requests. Rather than marking unprocessed checks as errored on shutdown, it does it now when starting up, in the `checkLastCreated` function (maybe we should change the name of the function). There, since we are already looping over the existing checks, we can check if some check has not been processed before.